desc/geometry/curve.py

```diff
@@ -858,6 +866,341 @@ def from_values(cls, coords, N=10, basis="xyz", name=""):
         )
 
 
+class FourierXYCurve(Curve):
+    """Curve that lies in a plane.
+
+    Parameterized by a point (the center of the curve), a vector (normal to the plane),
+    and Fourier series defining the X and Y coordinates in the plane.
+
+    Parameters
+    ----------
+    center : array-like, shape(3,)
+        Coordinates of center of curve, in system determined by basis.
+    normal : array-like, shape(3,)
+        Components of normal vector to planar surface, in system determined by basis.
+    X_n : array-like
+        Fourier coefficients of the X coordinate in the plane.
+    Y_n : array-like
+        Fourier coefficients of the Y coordinate in the plane.
+    modes : array-like
+        mode numbers associated with X_n and Y_n.
```

Comment:
can you say the default assumption is [-X_n.size//2 , X_n.size//2], with the n=0 mode included even tho it will be taken out later?

Reply:
FYI the default value for `modes` now handles both cases: 

- If `len(X_n)` is odd, then it assumes the n=0 mode is included like the other curve classes but gets ignored. 
- If `len(X_n)` is even, then it assumes the n=0 mode is not included since it is unused for this class.